addons/website_livechat/models/website_visitor.py

```diff
@@ -18,6 +19,12 @@ class WebsiteVisitor(models.Model):
                                        string="Visitor's livechat channels", readonly=True)
     session_count = fields.Integer('# Sessions', compute="_compute_session_count")
 
+    def _auto_init(self):
+        if not column_exists(self.env.cr, "website_visitor", "livechat_operator_id"):
+            self.env.cr.execute("ALTER TABLE website_visitor ADD COLUMN livechat_operator_id BOOLEAN DEFAULT false")
```

Comment:
Use `odoo.tools.sql.create_column` instead of manually executing an ALTER TABLE query. See https://github.com/odoo/odoo/blob/16.0/addons/l10n_latam_invoice_document/models/account_move_line.py for instance.

Reply:
Thx for the input! Should I amend the commit and reimplement with `create_column` ? First time submitting a PR, so I don't know the good practices yet.

Reply:
No you can just make your change locally. Then you add/commit them with whatever message you want. Then you manually rebase the commits to squash the new one into the first one and you keep the first commit message. After that you force push to this branch on dev.

Reply:
Ping me on discord if you want more explanations and a sequence of steps.

Reply:
Followed your description and I think it worked as expected. If not, do not hesitate to point it out.